openpype/tools/publisher/control.py

```diff
@@ -1589,20 +1589,19 @@ class PublisherController(BasePublisherController):
     Handle both creation and publishing parts.
 
     Args:
-        dbcon (AvalonMongoDB): Connection to mongo with context.
         headless (bool): Headless publishing. ATM not implemented or used.
     """
 
     _log = None
 
-    def __init__(self, dbcon=None, headless=False):
```

Comment:
Why is `dbcon` removed/redundant? Where is the connection/context of this particular context based on instead?

Reply:
This is just removing unnecessary middle-man. Connection from `dbcon` is not used at all and context is on `CreateContext`.